pr_agent/tools/pr_description.py

```diff
@@ -74,10 +74,11 @@ async def run(self):
 
             await retry_with_fallback_models(self._prepare_prediction)
 
-            get_logger().info(f"Preparing answer {self.pr_id}")
+            get_logger().info("Preparing description regarding", extra={"pr_id": self.pr_id})
```

Comment:
**Suggestion:** Use structured logging instead of string concatenation for better log message formatting and performance.
```suggestion
            get_logger().info("Preparing description regarding PR", extra={"pr_id": self.pr_id})
```